app/metrics.yaml

```diff
@@ -540,6 +540,22 @@ events:
     metadata:
       tags:
         - Sharing
+  form_data_failure:
+    type: event
+    description: |
+      Form data retrieval from a tab failed.
+    bugs:
+      - https://github.com/mozilla-mobile/fenix/issues/28280
+    data_reviews:
+      - https://github.com/mozilla-mobile/fenix/issues/28280#issuecomment-1376040252
+    notification_emails:
+      - android-probes@mozilla.com
+    data_sensitivity:
+      - technical
+    expires: 121
+    metadata:
+      tags:
+        - Sharing
```

Comment:
nit: @calumozilla This doesn't really fall under the "Sharing" tag which was meant for things like the share screen to third party apps.

When we get a chance, we should update this to maybe "Performance"?

Reply:
Ah, makes sense, I updated it in a new pr: https://github.com/mozilla-mobile/fenix/pull/28611